tests/repository_data/client/targets_map.json

```diff
@@ -0,0 +1,16 @@
+{
+ "targets_filename": "role1",
+ "keys":{
+  "c8022fa1e9b9cb239a6b362bbdffa9649e61ad2cb699d2e4bc4fdf7930a0e64a": {
+   "keyid_hash_algorithms": [
+    "sha256",
+    "sha512"
+   ],
+   "keytype": "ed25519",
+   "keyval": {
+    "public": "fcf224e55fa226056adf113ef1eb3d55e308b75b321c8c8316999d8c4fd9e0d9"
+   },
+   "scheme": "ed25519"
+  }
+ }
+}
```

Comment:
Is there an agreement about the targets map metadata format? AFAIR TAP 12 still misses the definition.

Reply:
There isn't. I based this format on a suggestion from the comments of the TAP 13 pr. It would probably be good to get some more consensus about this or a different format.

Reply:
cc @joshuagl, @trishankatdatadog, @JustinCappos

Reply:
Some of the other metadata files refer to other metadata files as well: they use the full name "targets.json" so `targets_filename` should probably do that too

Reply:
in the intervening 1 year we've discovered multiple reasons _not_ to use the full filename.